domains/weesp/prod.config.json

```diff
@@ -9,9 +9,13 @@
     "siteAddress": "meldingenweesp.amsterdam.nl",
     "siteTitle": "Meldingen Weesp",
     "title": "Meldingen Weesp",
-    "urgentContactInfo": "Voor spoedeisende zaken kunt u ook telefonisch contact opnemen met (0294) 491 391."
+    "urgentContactInfo": "Voor spoedeisende zaken kunt u ook telefonisch contact opnemen met (0294) 491 391.",
+    "logoDescription": "Logo gemeente Weesp, naar amsterdam.nl"
```

Comment:
Changing links.home (line 17) to `"https://www.weesp.nl"` means this should read `Logo gemeente Weesp, naar weesp.nl`, right?

Reply:
Correct! Good catch!